roles/rclone/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Uninstall current version of rclone
+  become: yes
+  shell: rm /usr/bin/rclone /usr/local/share/man/man1/rclone.1 '{{ rclone_config_location }}/rclone.conf'
+  ignore_errors: yes
+
+- name: Install rclone
+  shell: curl https://rclone.org/install.sh | sudo bash
```

Comment:
It looks like rclone is provided in the Ubuntu repositories and in the EPEL repository for RHEL systems.

Instead of using the install script with `sudo bash` (which may be unstable for future installs), can you install from the repository versions instead?